### ISSUE TITLE
--fastaout_rev write Read1 sequences instead of Read2

### ISSUE THREAD

**Issue by ZeweiSong** [torognes/vsearch#387]
This is what I tested:

```
vsearch v2.14.0_linux_x86_64, 15.8GB RAM, 6 cores
https://github.com/torognes/vsearch

Reading input file 100%
1 sequences kept (of which 0 truncated), 0 sequences discarded.
raw Read1
@CL100076083L1C001R002_470782/1
TTGCCCGATCCCTAGTCGGCATAGTTTACTGTTAAGACTACAACGGTATCTGATCGTTTTCGATCCCCTAACCTTCGTTCTTGATTAATGAAAACATCCT
+
=BB@>AB6:CCCBFADC?>@FBDBFFFECFADDFE?DEFFCEAE=BDBADAF0?EF@EA8FE<EFFFFFFFFF@EF8FFF>E;CFEGFFF@FFFEFDFGF
QCed Read1 FQ
@CL100076083L1C001R002_470782/1
TTGCCCGATCCCTAGTCGGCATAGTTTACTGTTAAGACTACAACGGTATCTGATCGTTTTCGATCCCCTAACCTTCGTTCTTGATTAATGAAAACATCCT
+
=BB@>AB6:CCCBFADC?>@FBDBFFFECFADDFE?DEFFCEAE=BDBADAF0?EF@EA8FE<EFFFFFFFFF@EF8FFF>E;CFEGFFF@FFFEFDFGF
QCed Read1 FA
>CL100076083L1C001R002_470782/1
TTGCCCGATCCCTAGTCGGCATAGTTTACTGTTAAGACTACAACGGTATCTGATCGTTTTCGATCCCCTAACCTTCGTTCTTGATTAATGAAAACATCCT
raw Read2
@CL100076083L1C001R002_470782/2
GTTGGGGGCATTGGTATTGAGTCGCTAGAGGTGAAATTCTTGGATTGACTCAAGACCAACTACTGCGAAAGCATTTGCCAAGGATGTTTTCATTAATCAA
+
FFFGFFFFFFFFFFFFFF@;GFFGF<GFFFFFFFFFFFF>FFFFFGFEFFFGEFDFFFFFFFFFGFF?FFFFFFFFFGFGFFFA<FFFGFFFGFFFFFFG
QCed Read2 FQ
@CL100076083L1C001R002_470782/2
GTTGGGGGCATTGGTATTGAGTCGCTAGAGGTGAAATTCTTGGATTGACTCAAGACCAACTACTGCGAAAGCATTTGCCAAGGATGTTTTCATTAATCAA
+
FFFGFFFFFFFFFFFFFF@;GFFGF<GFFFFFFFFFFFF>FFFFFGFEFFFGEFDFFFFFFFFFGFF?FFFFFFFFFGFGFFFA<FFFGFFFGFFFFFFG
QCed Read2 FA  <--- turned out to be Read1 sequence!
>CL100076083L1C001R002_470782/2
TTGCCCGATCCCTAGTCGGCATAGTTTACTGTTAAGACTACAACGGTATCTGATCGTTTTCGATCCCCTAACCTTCGTTCTTGATTAATGAAAACATCCT
```

**Comment by torognes** [comment]
Thank you for reporting this bug. I have identified the error in the code and will fix it and release a new version.

**Comment by ZeweiSong** [comment]
Cheers!

**Comment by torognes** [comment]
Reopening issue. Will close it when it has been resolved and released.

**Comment by torognes** [comment]
Fixed in commit a5af542391b6d3d8a92560a8232207f698331a2a.

**Comment by torognes** [comment]
Fixed in version 2.14.1 just released.
